src/pages/about-koselugo.astro

```diff
@@ -1,9 +1,82 @@
+            <h4 class="about-koselugo__title about-koselugo__title--only-tablet">Do you have questions about Koselugo? Talk to your child's doctor and download our</h4>
+            <a href="https://koselugo.com/faqs" class="button button--basic">Visit the FAQs here</a>
+            <a href="tel:+1888-765-4747" class="button button--basic button--only-mobile">Visit the FAQs here</a>
+          </div>
+        </Section>
+        <Section>
+          <Fragment slot="section-title">
+            CAN KOSELUGO HELP <span class="color-yellow">SHRINK PLEXIFORM TUMORS?</span>
+          </Fragment>
+          <div class="wrapper about-koselugo__clinical-study">
+            <h6 class="about-koselugo__title">KOSELUGO HAS BEEN SHOWN TO SHRINK PLEXIFORM TUMORS IN CHILDREN AGED 2 YEARS AND OLDER</h6>
+            <p>In a clinical study, Koselugo was proven to work for children with NF1 who have plexiform neurofibromas (PN) that cannot be completely removed by surgery:</p>
+            <div class="about-koselugo__container about-koselugo__container--flex">
+              <img class="about-koselugo__image" src="/assets/images/about-koselugo/66-percent-pie-chart.jpg" alt=" ">
+              <div class="about-koselugo__container about-koselugo__container--align-center">
+                <h6 class="about-koselugo__title">66% or 33 of 50 patients in the clinical study saw their plexiform neurofibromas (PN) shrink*</h6>
+                <p>82% or 27 of 33 patients who responded continued to benefit for at least one year.</p>
+              </div>
+            </div>
+          </div>
+        </Section>
+        <Section noHeader>
+          <div class="wrapper about-koselugo__journeys">
+            <div class="about-koselugo__container about-koselugo__container--clinical-study about-koselugo__container--visit">
+              <h4 class="about-koselugo__title">What did people taking Koselugo treatment experience in a clinical study?</h4>
+              <a href="https://koselugo.com/dosing-schedule#track-their-treatment" class="button button--basic">See their journeys</a>
+            </div>
+            <div class="about-koselugo__container about-koselugo__container--footnote">
+              <p>*Patients saw a 20% or more decrease in the size of their plexiform neurofibromas, confirmed by 3D magnetic resonance imaging scan.</p>
+              <p>NF1=neurofibromatosis type 1.</p>
+            </div>
+          </div>
+        </Section>
+        <Section noHeader>
+          <div class="wrapper about-koselugo__support-program">
+            <h6 class="about-koselugo__title">IS THERE A SUPPORT PROGRAM TO HELP MY CHILD ACCESS TREATMENT?</h6>
+            <h6 class="about-koselugo__title">OneSource™ Support Services</h6>
+            <div class="wrapper">
+              <div class="about-koselugo about-koselugo__container--flex">
+                <img class="about-koselugo__image" src="/assets/images/about-koselugo/personal-support-icon.png" alt=" ">
```

Comment:
This asset needs max-width. Also this content can appears in two columns since 700px
![image](https://user-images.githubusercontent.com/31933833/228382463-558aa38f-9c0d-43c6-9484-1a065c2999d9.png)